packages/office-addin-manifest/src/export.ts

```diff
@@ -0,0 +1,36 @@
+import * as fs from "fs";
+import * as fsExtra from "fs-extra";
+import * as jszip from "jszip";
+import * as path from "path";
+
+export function exportMetadataPackage(
+  output: string = "package/manifest.zip",
+  manifest: string = "manifest.json",
+  assets: string = "assets"
+): string {
+  const zip = new jszip();
+
+  if (fs.existsSync(manifest)) {
+    const manifestData = fs.readFileSync(manifest);
+    zip.file("manifest.json", manifestData);
+  } else {
+    throw new Error(`The file '${manifest}' does not exist`);
+  }
+
+  if (fs.existsSync(assets)) {
+    const files: string[] = fs.readdirSync(assets);
+    zip.folder(assets);
+    files.forEach((element) => {
+      const filePath = assets + "/" + element;
+      const fileData = fs.readFileSync(filePath);
+      zip.file(filePath, fileData);
+    });
+  } else {
+    throw new Error("Need folder of assets referenced by manifest file");
+  }
+
+  fsExtra.ensureDirSync(path.dirname(output));
+  zip.generateNodeStream({ type: "nodebuffer", streamFiles: true }).pipe(fs.createWriteStream(output));
+
+  return output;
```

Comment:
I don't think we need to return anything. 
Can we just make this function `void`?

Reply:
I started with a void, but thought returning the actual path used could be useful and I made use of it in the test case.

Reply:
But the default case is likely to be the most common one.  It might be interesting to return the whole path or a path object instead.

Reply:
As I don't think this is a big deal, we can leave it returning the path like it is.